itou/www/siae_evaluations_views/tests/tests_institutions_views.py

```diff
@@ -2954,7 +2954,7 @@ def test_num_queries_in_view(self):
             + 6  # fetch evaluated_siae and its prefetch_related
             + 1  # one again institution membership
             + 3  # savepoint, update session, release savepoint
-            + 5  # issue with evaluated_job_application.evaluated_siae.state
+            + 3  # issue with evaluated_job_application.evaluated_siae.state
```

Comment:
Je cherche pourquoi on charge deux fois la job application
```suggestion
            + 1  # evaluated_job_application
            + 1  # evaluated_administrative_criteria
            + 1  # evaluated_job_application
            + 3  # savepoint, update session, release savepoint
```

Reply:
Ok, j’ai compris. On précharge les données sur l’objet `job_application`, et la SIAE est retrouvée avec `job_application.evaluated_siae`, ce qui charge un nouvel objet sans cache de relations. Évidemment, à l’appel à `evaluated_siae.state`, on recharge les candidatures et les critères administratifs.

Reply:
Le refactor semble réduire le soucis :)